examples/multi_gpu/papers100m_gcn.py

```diff
@@ -1,142 +1,331 @@
+import argparse
 import os
+import tempfile
 import time
 
+import numpy as np
 import torch
 import torch.distributed as dist
 import torch.multiprocessing as mp
 import torch.nn.functional as F
 from ogb.nodeproppred import PygNodePropPredDataset
+from torch.distributed.algorithms.join import Join
 from torch.nn.parallel import DistributedDataParallel
+from torchmetrics import Accuracy
 
-from torch_geometric.loader import NeighborLoader
-from torch_geometric.nn import GCNConv
+import torch_geometric
 
+os.environ['CUDF_SPILL'] = '1'
```

Comment:
It might be worth adding some comment like this:
```suggestion
# Allow computation on objects that are larger than GPU memory
# https://docs.rapids.ai/api/cudf/stable/developer_guide/library_design/#spilling-to-host-memory
os.environ['CUDF_SPILL'] = '1'
```